mlflow/utils/databricks_utils.py

```diff
@@ -583,7 +583,6 @@ def get_databricks_host_creds(server_uri=None):
 
     .. Warning:: This API is deprecated. In the future it might be removed.
     """
-
```

Comment:
Probably safe to leave this in so its one less file to check. 🙂

Reply:
done 👍